processor/k8sprocessor/config.go

```diff
@@ -56,8 +56,9 @@ type ExtractConfig struct {
 	// The field accepts a list of strings.
 	//
 	// Metadata fields supported right now are,
-	//   namespace, podName, podUID, deployment, cluster, node and startTime
-	//
+	//   namespace, podName, podUID, deployment, cluster, node, startTime,
```

Comment:
We're keeping the old names for backward compat right? In that case I'd not document them anymore. 

I'd also add a deprecation warning if any of the old field names are used.

Reply:
Thanks for the input @owais . Yes, we are keeping them for backward compatibility. I will remove the docs and a deprecation warning. Also, should I remove references to other parts of the codebase which refer to these older fields and migrate them to newer ones?

Reply:
Yes, that'd be great. Thanks.